pkg/nodes/enricher/enricher_impl.go

```diff
@@ -72,12 +78,15 @@ func (e *enricherImpl) EnrichNodeWithInventory(node *storage.Node, nodeInventory
 
 // EnrichNode enriches a node with the integration set present.
 func (e *enricherImpl) EnrichNode(node *storage.Node) error {
-	return e.EnrichNodeWithInventory(node, nil)
+	return e.EnrichNodeWithInventory(node, nil, nil)
 }
 
-func (e *enricherImpl) enrichWithScan(node *storage.Node, nodeInventory *storage.NodeInventory) error {
+func (e *enricherImpl) enrichWithScan(node *storage.Node, nodeInventory *storage.NodeInventory, indexReport *v4.IndexReport) error {
 	errorList := errorhelpers.NewErrorList(fmt.Sprintf("error scanning node %s:%s", node.GetClusterName(), node.GetName()))
 
+	log.Debugf("Enriching Node with Inventory: %v / Index: %v", nodeInventory.GetNodeId() != "", indexReport.GetState() != "")
```

Comment:
```suggestion
	log.Debugf("Enriching Node with Inventory: %t / Index: %t", nodeInventory.GetNodeId() != "", indexReport.GetState() != "")
```

Reply:
The comment is resolved, but the suggestion seem not to be applied. Could you check again?